torch/autograd/gradcheck.py

```diff
@@ -255,7 +256,9 @@ def get_stride(size):
 def _get_numerical_jacobian(
     fn, inputs, outputs=None, target=None, eps=1e-3, is_forward_ad=False
 ) -> List[Tuple[torch.Tensor, ...]]:
-    """Computes the numerical Jacobian of `fn(inputs)` with respect to `target`. If
+    """Calculate numerical Jacobian.
```

Comment:
Prefer the original first sentence.

Reply:
Alright, I have made the changes. Please review this.